docs/deploying_apps/deploying_django.md

```diff
@@ -101,23 +100,23 @@ You can now view your app at `https://APPNAME.cloudapps.digital`.
 
 ## PostgreSQL setup with Django
 
-In your `settings.py` file, make sure you import the package we added to the `requirements.txt` file above:
+Add this line to your ``requirements.txt``:
```

Comment:
Add "This will install the PostgreSQL driver"
Also, the line `dj-database-url==0.3.0 #grabs environment variables and dumps them into a Django settings file` above could be moved from above to here as it is only relevant here. It also makes more sense with the following line `make sure you import the package we added to the `requirements.txt` file above`.